setup.py

```diff
@@ -67,8 +67,11 @@ def get_version(fname):
     description="Python library and CLI for Speechmatics",
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    install_requires=read_list("requirements.txt"),
+    install_requires=[req for req in read_list("requirements.txt") if not req.startswith("pyannote")],
     tests_require=read_list("requirements-dev.txt"),
+    extras_require={
+        "metrics": ["pyannote.core", "pyannote.database"],
```

Comment:
Hey @olgeni 👋 thanks so much for this PR!

Moving optional dependencies to extras_require is a great improvement.

One small suggestion: could you update the extras_require section to include the rest of the dependencies used by asr_metrics? Right now, a few are missing:

```
jiwer
regex
more-itertools
docopt
tabulate>=0.8.9
```


Once that's updated, I think this will be good to go!

Reply:
You can probably add the asr_metrics dependencies to a requirements-metrics.txt file and remove them from the main requirements file, and error handling for missing optional dependencies.